src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const rentForDay = 40;
```

Comment:
Consider using a constant with a descriptive name for the daily rental cost to adhere to the checklist item 2 regarding not using literals in expressions.